File: test/data/src/py/PhylSowingDateCorrection.py

if (latitude < 0):
    if (sowingDay > sDsa_sh):
        fixPhyll = p * (1 - rp * min(sowingDay - sDsa_sh, sDws))
    else: fixPhyll = p
else:
    if (sowingDay < sDsa_nh):
        fixPhyll = p * (1 - rp * min(sowingDay, sDws))
    else: fixPhyll = p

File: test/data/phenology/crop2ml/algo/java/PhylSowingDateCorrection.java

if (latitude < 0.0)
{
	if (sowingDay > sDsa_sh)
	{
		fixPhyll = p * (1 - rp * Math.min(sowingDay - sDsa_sh, sDws));
	}
	else fixPhyll = p;
}
else
{
	if (sowingDay < sDsa_nh)
	{
		fixPhyll = p * (1 - rp * Math.min(sowingDay, sDws));
	}
	else fixPhyll = p;
}